docs/examples/llama_datasets/downloading_llama_datasets.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>Yup, that's correct :) </p><p> </p><p><code>RagExamplePrediction</code> contains <code>response</code> and <code>contexts</code> . So hopefully it can be understood that the <code>response</code> is the generated response for this prediction and the <code>contexts</code> is its associated retrieved contexts.</p><p> </p><p>If we can make this more clear I'd be happy to adjust.</p>